internal/report.go

```diff
@@ -396,6 +396,9 @@ func GenerateSummary(conv *Conv, r []tableReport, badWrites map[string]int64) st
 		warnings += t.Warnings * weight
 		if t.SyntheticPKey != "" {
 			missingPKey = true
+			if conv.MigrationData.GetSchemaPatterns().GetMissingPrimaryKey() {
+				conv.MigrationData.SchemaPatterns.MissingPrimaryKey = &missingPKey
```

Comment:
We are updating the bool value for each table here. We should maintain a count for MissingPrimaryKey instead. So, it should be conv.MigrationData.SchemaPatterns.MissingPrimaryKey++ here.